Changelog.md

```diff
@@ -6,6 +6,10 @@
 - Increase China coverage by adding tests for special extra-holidays & extra-working days cases.
 - Added compatibility with Python 3.8 (#406).
 
+## 7.1.0 (2019-11-15)
```

Comment:
As it's mentioned in the Pull-Request template, you must not change the release number here. When you're adding your changelog item, you simple add it to the current "master (unreleased)" section. It's the maintainers duty to handle releases and version numbers.